browser/templates/browser/person.html

```diff
@@ -261,11 +308,133 @@ <h3>Locations</h3>
     </div>
     {% endif %}
     {% endwith %}
+
+    <!-- Positions -->
+    {% with person|get_positions as positions %}
+    {% if positions|length > 0 %}
+    <div role="tabpanel" class="tab-pane" id="person-positions">
+        <div class="panel panel-mbl">
+            <div class="panel-heading">
+                <h3>Positions</h3>
+            </div>
+            <div class="container">
+            <label for="dropdownTest" >Sort by Role:</label>
+            <select id="dropdownTest" name="cars">
```

Comment:
why is this called 'dropdownTest'? Why is the name 'cars'?

Reply:
Changed the variable names